middlewares/middlewares.go

```diff
@@ -127,3 +127,55 @@ func Logging(closures ...func(*http.Request) []zapcore.Field) Middleware {
 		})
 	}
 }
+
+func NewStandardZapLevelConfig(level zapcore.Level) (zap.Config) {
+	return zap.Config{
+		Level:       zap.NewAtomicLevelAt(level),
+		Development: false,
```

Comment:
What does this do? Do you think it is something we should hookup to an env var if we need to debug or something?

Reply:
https://github.com/uber-go/zap/tree/master/config.go#L53 
    // Development puts the logger in development mode, which changes the
    // behavior of DPanicLevel and takes stacktraces more liberally.
And then under DisableStacktrace:
    // DisableStacktrace completely disables automatic stacktrace capturing. By
    // default, stacktraces are captured for WarnLevel and above logs in
    // development and ErrorLevel and above in production.
So, "Development" means we "autmatically" get stacktraces on Warn messages as well as Error and Fatal. I think we want it to stay false. It's not worth complicating the log setup code for a probable YAGNI.